need to let readers know that 215 needs to be replaced with the clien…

rjaitly opened the pull request:
…t id

Please note that the value 215 in 
https://contoso.filesanywhere.com/saml20.aspx?c=215
and
https://<company name>.filesanywhere.com/saml20.aspx?c=215
is a clientid and is just an example. It needs to be replaced with the actual value.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rjaitly committed:
need to let readers know that 215 needs to be replaced with the client id

Please note that the value 215 in 
https://contoso.filesanywhere.com/saml20.aspx?c=215
and
https://<company name>.filesanywhere.com/saml20.aspx?c=215
is a clientid and is just an example. It needs to be replaced with the actual value.
